src/test/java/eu/dety/burp/joseph/attacks/invalid_curve/InvalidCurveTest.java

```diff
@@ -0,0 +1,24 @@
+package eu.dety.burp.joseph.attacks.invalid_curve;
+
+import burp.IBurpExtenderCallbacks;
+import eu.dety.burp.joseph.BurpExtenderCallbacksMock;
+import org.junit.BeforeClass;
+
+class InvalidCurveTest {
+    private static InvalidCurveInfo invalidCurveInfo;
+    private static InvalidCurve invalidCurve;
+    private static IBurpExtenderCallbacks callbacks;
+
+    @BeforeClass
+    public static void setUp() {
+        callbacks = new BurpExtenderCallbacksMock();
+        invalidCurveInfo = new InvalidCurveInfo(callbacks);
+        invalidCurve = new InvalidCurve(callbacks, invalidCurveInfo);
+    }
+
+    // @Test
+    // public void performAttackTest() {
+    // invalidCurve.performAttack();
```

Comment:
You can define a series of longer extended integration tests, which work only if specifically requested (because of their execution times)